src/query.js

```diff
@@ -49,11 +47,10 @@ export function isInvalidKey(key, s){
 export function sanitizeArray(key, val) {
   const arr = []
   let knownKeys
-
   if(key === 'msamds') knownKeys = msaKeys
   else if(key === 'states') knownKeys = stateKeys
   else if(key === 'counties') knownKeys = countyKeys
-  else if(key === 'leis') knownKeys = leiKeys
+  else if(key === 'leis') return val
```

Comment:
While I think it's still likely possible to sanitize the array based off the fetched data, it probably isn't really worth the trouble as that's more of a protection against someone trying to make the app act funny than anything important

Reply:
I had a comment on this line indicating that we sanitize in the Geography component but I deleted it. 🤦‍♂  We do still sanitize the LEIs once we have fetched data:

https://github.com/cfpb/hmda-data-browser/blob/filter-leis-by-geography/src/geo/Geography.jsx#L77

https://github.com/cfpb/hmda-data-browser/blob/filter-leis-by-geography/src/geo/Geography.jsx#L86

https://github.com/cfpb/hmda-data-browser/blob/filter-leis-by-geography/src/geo/leiUtils.js#L8-L15